frontend/src/metabase/admin/settings/components/SettingsLdapForm.jsx

```diff
@@ -47,7 +48,13 @@ export default class SettingsLdapForm extends React.Component {
           },
           {
             title: t`Group Schema`,
-            settings: ["ldap-group-sync", "ldap-group-base"],
+            settings: [
+              "ldap-group-sync",
+              "ldap-group-base",
+              "ldap-sync-admin-group" in this.props.settingValues
+                ? "ldap-sync-admin-group"
+                : null,
+            ],
```

Comment:
```suggestion
            ].filter(Boolean),
```
I don't see any filtering done in `SettingsBatchForm` so probably safest to add a `filter(Boolean)` here to remove the potentially falsy value from this array.